process_test.go

```diff
@@ -166,3 +173,171 @@ func TestProcessImplementations(t *testing.T) {
 		})
 	}
 }
+
+// Returns path to release and to mongod
+func downloadMongodb(t *testing.T) (string, string) {
```

Comment:
maybe move these tests and their helpers to a different file. totally optional

Reply:
Done